### PR TITLE
T-Echo display support

### PR DESCRIPTION
**Pull request by jeremybox**
This adds display support for the lilygo t-echo device.

The graphics aren't perfect, and still need some work, as the resolution is wrong, but it's reasonably usable as committed.

I'll keep working on it in the background, let me know if you want to wait until it's "perfect"

### CODE REVIEW

File: Makefile
Comment (jeremybox) [inline Makefile:1]: this minor change just adds some compatibility when i was testing/building on a mac, but I notice we may have previously also committed my directory structure (../Reticulum/RNS/ for the upstream RNS repo), so we may or may not want to revert that change.

**Comment by jacobeva** [comment]
Let's keep this PR open until it's properly finished :)
Thank you for the hard work!

**Comment by jeremybox** [comment]
With that, there's going to be more commits, and I'm still bad at squashing them hah, but we can figure it out.

**Comment by jacobeva** [comment]
No problem, we can cross that bridge when we come to it mate!

**Comment by jeremybox** [comment]
All that remains is scaling the graphics, which I don't know if I'll have time to do or not in the next week or two.  I recommend committing as is, as the graphics are no longer garbled, and are readable and usable now but simply don't fill the entire screen.

**Comment by jacobeva** [comment]
Okay, I will merge this when I have time. Thank you for the hard work!

**Comment by jeremybox** [comment]
Also, during my testing, I verified that all of my changes did not break a heltechv3 device's graphics, which used the default/standard oled stuff.  Feel free to test with other devices, but this makes me reasonably confident that none of my changes to the Display.h code will impact any other devices.

**Comment by jacobeva** [comment]
Thank you, that is very helpful.

**Comment by jacobeva** [comment]
GH is being weird and not marking this as merged. It's actually been merged just now in afce928149db10f52ca00189b2ef052bf6f2c79e

**Pull request closed without merging**